src/App.js

```diff
@@ -1,28 +1,50 @@
-import logo from './logo.svg';
-import './App.css';
+// import logo from './logo.svg';
+// import './App.css';
 
+// function App() {
+//   return (
+//     <div className="App">
+//       <header className="App-header">
+//         <img src={logo} className="App-logo" alt="logo" />
+//         <p>
+//           Edit
+//           {' '}
+//           <code>src/App.js</code>
+//           {' '}
+//           and save to reload.
+//         </p>
+//         <a
+//           className="App-link"
+//           href="https://reactjs.org"
+//           target="_blank"
+//           rel="noopener noreferrer"
+//         >
+//           Learn React
+//         </a>
+//       </header>
+//     </div>
+//   );
+// }
+
+// export default App;
```

Comment:
- [ ] [OPTIONAL]
There are commented code blocks found in your code on the repository. Kindly endeavor to remove **all** commented code from your code so as to keep your code well organized.